Sources/MongoSwift/Operations/FindOperation.swift

```diff
@@ -147,6 +151,7 @@ public struct FindOptions: Codable {
         comment: String? = nil,
         cursorType: MongoCursorType? = nil,
         hint: IndexHint? = nil,
+        `let`: BSONDocument? = nil,
```

Comment:
I guess because its not specifying a type, the original error is this:
`error: Colon Spacing Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)`